awx/ui/src/components/SelectedList/DraggableSelectedList.js

```diff
@@ -21,114 +17,94 @@ const RemoveActionSection = styled(DataListAction)`
-    );
+  const removeItem = (item) => {
+    onRemove(selected.find((i) => i.name === item));
   };
 
-  const onDragCancel = () => {
-    setLiveText(t`Dragging cancelled. List is unchanged.`);
-    setIsDragging(false);
-  };
+  function reorder(list, startIndex, endIndex) {
+    const result = Array.from(list);
+    const [removed] = result.splice(startIndex, 1);
+    result.splice(endIndex, 0, removed);
+    return result;
+  }
 
-  const onDragFinish = (newItemOrder) => {
-    const selectedItems = newItemOrder.map((item) =>
-      selected.find((i) => i.name === item)
-    );
-    onRowDrag(selectedItems);
-    setIsDragging(false);
-  };
+  function dragItem(item, dest) {
+    if (!dest || item.index === dest.index) {
+      return false;
+    }
 
-  const removeItem = (item) => {
-    onRemove(selected.find((i) => i.name === item));
-  };
+    const newItems = reorder(selected, item.index, dest.index);
+    onRowDrag(newItems);
+    return true;
+  }
 
   if (selected.length <= 0) {
     return null;
   }
 
-  const orderedList = selected.map((item) => item?.name);
-
   return (
     <>
-      <DataList
-        aria-label={t`Draggable list to reorder and remove selected items.`}
-        data-cy="draggable-list"
-        itemOrder={orderedList}
-        onDragCancel={onDragCancel}
-        onDragFinish={onDragFinish}
-        onDragMove={onDragMove}
-        onDragStart={onDragStart}
-      >
-        {orderedList.map((label, index) => {
-          const rowPosition = index + 1;
-          return (
-            <DataListItem id={label} key={rowPosition}>
-              <DataListItemRow>
-                <DataListControl>
-                  <DataListDragButton
-                    aria-label={t`Reorder`}
-                    aria-labelledby={rowPosition}
-                    aria-describedby={t`Press space or enter to begin dragging,
-                    and use the arrow keys to navigate up or down.
-                    Press enter to confirm the drag, or any other key to
-                    cancel the drag operation.`}
-                    aria-pressed="false"
-                    data-cy={`reorder-${label}`}
-                    isDisabled={selected.length === 1}
-                  />
-                </DataListControl>
-                <DataListItemCells
-                  dataListCells={[
-                    <DataListCell key={label}>
-                      <span id={rowPosition}>{`${rowPosition}. ${label}`}</span>
-                    </DataListCell>,
-                  ]}
-                />
-                <RemoveActionSection aria-label={t`Actions`} id={rowPosition}>
+      <DragDrop onDrop={dragItem}>
+        <Droppable>
+          {selected.map(({ name: label, id }, index) => {
+            const rowPosition = index + 1;
+            return selected.length < 2 ? (
+              <div key={rowPosition}>
+                <DragWrapper disabled>
+                  <GripVerticalIcon />
```

Comment:
On the previous implementation once the user hover over this button - we allow the user to drag the row.

There were lines between rows, and the `x` to remove that row was positioned on the end of the row. This change may break visual tests..

Current implementation:

![image](https://user-images.githubusercontent.com/9053044/144502727-e2ff5de7-a180-476a-a829-dec7710adeeb.png)

Current devel implementation.

<img width="1229" alt="image" src="https://user-images.githubusercontent.com/9053044/144502827-05c7ead2-09e3-43b4-9143-78c92136de04.png">